E3Next/Processors/Rez.cs

```diff
@@ -401,11 +392,9 @@ private static void MultiRez(RezType rezType = RezType.AE)
 
                     //assume consent was given
                     MQ.Cmd("/corpse");
-
-               
-                    foreach (var spell in _currentRezSpells)
+                    
+                    foreach (var spell in E3.CharacterSettings.HealResurrection)
```

Comment:
In this loop, if it's a rez token, validate we have one

Reply:
I believe this is already handled via a combination of the constructor for `Data.Spell` which will populate the initial item data from MQ queries on first initialization and `Casting.CheckReady` which returns true only when `Me.ItemReady[<name>]` evals to true.

https://github.com/RekkasGit/E3Next/blob/fdc16a033bc6f067692b6078d8c79102a5b35dc2/E3Next/Processors/Casting.cs#L894

Reply:
Ah I think you're right. Good call out.

Reply:
I need to look at this closer, but I believe you will DC from the Wait4Rez tell being spammed, as yes the checkready may fail, but you need to kickout long before you reach that point.  This was the reason for the Refresh Spell method.

Reply:
You might be right there, I'll look into it there is a clean way to throttle the attempts

Reply:
Less about throttle, more of a check if its in the spell book, if not, check if its an item.. if not (we are out of tokens), skip it.